### PR TITLE
Release Blog Post for ADOT collector  v0.26.1 and v0.25.1

### PR DESCRIPTION
**Pull request by vasireddy99**
This PR is created to add the release blog post for v0.26.1 and v0.25.1

### CODE REVIEW

File: src/content/BlogPosts/blogPosts.yaml [hunk 1]
```diff
@@ -6,6 +6,14 @@ description:
 path: /blog
 
 blogs:
+  - title: "AWS Distro for OpenTelemetry v0.26.0 is now available"
```
Comment (bryan-aguilar) [inline src/content/BlogPosts/blogPosts.yaml:9]:
```suggestion
  - title: "AWS Distro for OpenTelemetry v0.26.1 is now available"
```

File: src/content/BlogPosts/blogPosts.yaml [hunk 2]
```diff
@@ -6,6 +6,14 @@ description:
...
+    author: "Pavan Sai Vasireddy"
+    date: "16-Feb-2023"
+    body:
+      "AWS Distro for OpenTelemetry v0.26.0 is now available. You can download the latest AWS Distro for OpenTelemetry Collector image
```
Comment (bryan-aguilar) [inline src/content/BlogPosts/blogPosts.yaml:13]:
```suggestion
      "AWS Distro for OpenTelemetry v0.26.1 is now available. You can download the latest AWS Distro for OpenTelemetry Collector image
```

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.26.1.mdx [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+title: "AWS Distro for OpenTelemetry v0.26.1"
```
Comment (bryan-aguilar) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.26.1.mdx:2]: Is this blog post supposed to be for both versions?

Comment (vasireddy99) [reply src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.26.1.mdx]: yeah , have modified them to reflect the same for two versions

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.26.1.mdx [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
+title: "AWS Distro for OpenTelemetry v0.26.1 and v0.25.1 are"
```
Comment (bryan-aguilar) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.26.1.mdx:2]:
```suggestion
title: "AWS Distro for OpenTelemetry Collector v0.26.1 and v0.25.1"
```

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.26.1.mdx [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+description: This blog post is the release announcement for ADOT Collector v0.26.1 and v0.25.1 are
```
Comment (bryan-aguilar) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.26.1.mdx:3]:
```suggestion
description: This blog post is the release announcement for ADOT Collector v0.26.1 and v0.25.1
```

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.26.1.mdx [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+---
+
+import SectionSeparator from "components/MdxSectionSeparator/sectionSeparator.jsx";
+
+<SectionSeparator />
+
+[AWS Distro for OpenTelemetry (ADOT)](https://aws-otel.github.io/) v0.26.1 is now available.
```
Comment (bryan-aguilar) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.26.1.mdx:10]:
```suggestion
[AWS Distro for OpenTelemetry (ADOT)](https://aws-otel.github.io/) Collector v0.26.1 and v0.25.1 are now available.
```

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.26.1.mdx [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+ADOT Collector to at least v0.26.1 or v0.25.1 at the earliest opportunity. See https://aws-otel.github.io/docs/getting-started/collector for information on deploying the ADOT Collector.
+
+Reference: https://groups.google.com/g/golang-announce/c/V0aBFqaFs_E/m/CnYKgKwBBQAJ
+
+**Release Highlights**
+
+- Recompile with Go 1.19.6 to mitigate CVE-2022-41722, CVE-2022-41723 and CVE-2022-41724
+
+Detailed release notes are on [GitHub](https://github.com/aws-observability/aws-otel-collector/releases).
+All code changes are upstream in the respective OpenTelemetry project components.
```
Comment (bryan-aguilar) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.26.1.mdx:29]:
```suggestion
```

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.26.1.mdx [hunk 8]
```diff
@@ -0,0 +1,49 @@
+---
+title: "AWS Distro for OpenTelemetry v0.26.1 and v0.25.1"
```
Comment (bryan-aguilar) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.26.1.mdx:2]:
```suggestion
title: "AWS Distro for OpenTelemetry Collector v0.26.1 and v0.25.1"
```

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.26.1.mdx [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+description: This blog post is the release announcement for ADOT Collector v0.26.1 and v0.25.1
+---
+
+import SectionSeparator from "components/MdxSectionSeparator/sectionSeparator.jsx";
+
+<SectionSeparator />
+
+[AWS Distro for OpenTelemetry (ADOT)](https://aws-otel.github.io/) v0.26.1 and v0.25.1 are now available.
```
Comment (bryan-aguilar) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.26.1.mdx:10]:
```suggestion
[AWS Distro for OpenTelemetry (ADOT)](https://aws-otel.github.io/) Collector v0.26.1 and v0.25.1 are now available.
```

**Review by bryan-aguilar — COMMENTED** [review]
Please change the file name for the blog post to indicate that it represents both versions

**Review by bryan-aguilar — APPROVED** [review]

**Pull request merged**